webapp/src/dogma/features/file/FileList.tsx

```diff
@@ -11,9 +11,17 @@ export type FileListProps<Data extends object> = {
   data: Data[];
   projectName: string;
   repoName: string;
+  handleCopyApiUrl: Function;
+  handleCopyAsCurlCommand: Function;
```

Comment:
Let's add some tests for these. 😄 
Here's an example for `handleCopyApiUrl`.
1) Modify the following in `FileList.test.tsx`.
```
    expectedProps = {
      data: mockfileList,
      projectName: 'ProjectAlpha',
      repoName: 'repo1',
      handleCopyApiUrl: jest.fn(),
    };
```
2) Add the unit test
```
  it('calls handleCopyApiUrl when copy API URL button is clicked', () => {
    const { getAllByText } = render(<FileList {...expectedProps} />);
    const firstButton = getAllByText('Copy API URL', { selector: 'button' })[0];
    fireEvent.click(firstButton);
    expect(expectedProps.handleCopyApiUrl).toHaveBeenCalledTimes(1);
  });
```
3) Don't forget to import 
```import { fireEvent, render } from '@testing-library/react';```

4) Similarly for `handleCopyAsCurlCommand`

Reply:
on it @rainy789

Reply:
added ya @rainy789 

<img width="784" alt="image" src="https://user-images.githubusercontent.com/24659468/209423388-85b9e817-6328-4eca-9d86-b6736445b019.png">